.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -25,10 +25,10 @@ body:
       required: true
 
   - type: input
```

```suggestion
  - type: textarea
```
Could you change this to a `textarea` so that formatting doesn't get messed up when copy-pasting info?

You could also remove the Vercel version input below, now that it is included in the environment info.

---

ah... I thought it was already a textarea! :sweat: 

that's why all current issues have the info on one line :sweat: